org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/junit/Junit4Fragment2.xtend

```diff
@@ -82,19 +82,18 @@ class Junit4Fragment2 extends AbstractStubGeneratingFragment {
 		return fileAccessFactory.createXtendFile(exampleRuntimeTest, '''
 			@�runWith�(�xtextRunner�)
 			@�injectWith�(�injectorProvider�)
-			class �exampleRuntimeTest�{
-			
+			class �exampleRuntimeTest� {
 				@�Inject�
 				�parseHelper�<�rootType�> parseHelper
-			
-				@�test� 
+				
```

Comment:
Yes, it is. It is basically a newline between the injection and the test method. The tab is there in this new line with class-level indentation.